enterprise/ha/src/docs/dev/haproxy.asciidoc

```diff
@@ -80,6 +80,17 @@ Note that the servers in the `slave` backend are configured the same way as in t
 
 Then by putting all the above configurations into one file, we get a basic workable HAProxy configuration to perform load balancing for applications using the Bolt Protocol.
 
+By default, encryption is enabled between servers and drivers. With encryption turned on, the HAProxy configuration constructed above needs no change to work directly in TLS/SSL passthrough layout for HAProxy.
+However depending on the driver authentication strategy adopted, some special requirements might apply to the server certificates.
+
+For drivers using trust-on-first-use authentication strategy, each driver would register the HAProxy port it connects to with the first certificate received from the cluster.
+Then for all subsequent connections, the driver would only accept the connections from the server whose certificate is the same as the one registered.
```

Comment:
@zhenlineo driver _never, ever_ accepts connections _from_ server. Maybe use 'for' instead of 'from'?